client/app/src/main/java/com/example/mrides/Activity/HomePage.java

```diff
@@ -76,27 +83,27 @@ public void onClick(View view) {
     }
 
     public void createMatchedRoutes() {
-//        LinearLayout linearLayout = (LinearLayout) findViewById(R.id.layoutScrollMatch);
-//
-//        for (int i = 0; i < matchedRoutes.length; i++) {
-//            Button route = new Button(this);
-//            LinearLayout.LayoutParams params = styleButton(route);
-//            route.setText(matchedRoutes[i]);
-//            linearLayout.addView(route, params);
-//
-//        }
+        LinearLayout linearLayout = (LinearLayout) findViewById(R.id.layoutScrollMatch);
```

Comment:
Redundant code. Similar code below. Create a helper method for this block of code.